monkey/infection_monkey/payload/payload.py

```diff
@@ -0,0 +1,21 @@
+from abc import abstractmethod
+
+import infection_monkey.payload
+from infection_monkey.utils.plugins import Plugin
+
+
+class Payload(Plugin):
+    @staticmethod
+    def base_package_file():
```

Comment:
Can this be a property? What's the point of `base_` prefix? Can't this just be named `get_package_file`?

Reply:
As Payload is a plugin we have those methods in the Plugin Interface as methods not properties. The point of `base_`  ( maybe @shreyamalviya  can correct me ) as is stated:  `__file__` is the pathname of the file from which the module was loaded. Which is in this case the base of the plugin.